Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -244,6 +249,14 @@ extension TimestampFormat {
     public static let iso8601 = TimestampFormat("iso8601", formatter: ISO8601DateFormatter.init)
 }
 
+extension TimestampFormat {
+    public static let iso8601WithMilliseconds = TimestampFormat("iso8601WithMilliseconds", formatter: {
+        let formatter = ISO8601DateFormatter()
+        formatter.formatOptions.insert(.withFractionalSeconds)
+        return formatter
+    })
+}
```

Comment:
While I certainly agree that defaulting to GMT is correct, should there perhaps also be a way to specify `formatter.timeZone` without having to completely override these 8601 built-ins?

Reply:
Yes, probably worth a new issue.